builder/build-workstation-template

```diff
@@ -17,33 +16,18 @@ sd_template_dir="${build_dir}/qubes-template-securedrop-workstation"
 rm -rf "${qubes_builder_dir}" "${sd_template_dir}"
 
 git clone https://github.com/qubesos/qubes-builder "${build_dir}/qubes-builder"
-git clone git@github.com:freedomofpress/qubes-template-securedrop-workstation "${build_dir}/qubes-template-securedrop-workstation"
+git clone https://github.com/freedomofpress/qubes-template-securedrop-workstation "${sd_template_dir}"
 
-# Temporary! Related to unmerged PR logic
-cd "${build_dir}/qubes-template-securedrop-workstation"
-git checkout first_commit
+cd "${qubes_builder_dir}"
 
-cd "${build_dir}/qubes-builder"
-./setup # Appears this is required, worth confirming
+# Provision securedrop-workstation config so that we get the correct sources via `make get-sources`.
+cp ${sd_template_dir}/securedrop-workstation.conf builder.conf
 
 # Install dependencies from Qubes builder logic.
 make install-deps
-
-# These *should* be pulled in automatically, but currently aren't.
-# May be worth a PR targeting upstream.
-sudo dnf install -y perl-Digest-MD5 perl-Digest-SHA
```

Comment:
Removing this line causes failure for me! Can you try rebooting your VM and seeing if it's really not necessary? You may have the packages sticking around from a previous invocation of this line.

Reply:
I've retested this in a clean VM, and the dependency is not required. I have recently upgraded to fedora-28 templates. Given that f26 is end of life (see https://github.com/freedomofpress/securedrop-workstation/issues/114), I think we can safely leave this out of the script for the time being.

Reply:
> recently upgraded to fedora-28 templates

Ah, thanks for clarifying. Alright, let's punt to #114, then—if anyone reports this error, we'll know their templates are out of date. :slightly_smiling_face: